xml/vt_docker_quick.xml

```diff
@@ -328,7 +328,7 @@
    </para>
 <!--taroth 2015-07-02: http://doccomments.provo.novell.com/comments/28724 -->
    <procedure>
-    <title>Enabling the Container Module</title>
+    <title>Enabling the Container Module - interactive way</title>
```

Comment:
It's more about having to perform manual operation (hence interactive) vs no human interaction (non interactive).

You can add the containers module also from the command line using yast2, but that would require human interactions.

I'll fix the title case. Do you have better ideas about the value to put in there?